geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/GeoObjectDimensionValues.java

```diff
@@ -0,0 +1,143 @@
+package mil.nga.giat.geowave.analytics.tools;
+
+import java.nio.ByteBuffer;
+
+/**
+ * 
+ * Extracted numeric dimension values associated with an item or
+ * a sum of dimension values from multiple items.
+ * 
+ */
+public class GeoObjectDimensionValues
```

Comment:
is it worth making this implement Persistable and changing 'to/fromBytes' to 'to/fromBinary?'  If nothing else that is consistent with other persistable classes in GeoWave.